packages/react-reconciler/src/ReactFiberCommitWork.old.js

```diff
@@ -3023,14 +3165,16 @@ function commitTracingMarkerPassiveMountEffect(finishedWork: Fiber) {
     (instance.pendingBoundaries === null ||
       instance.pendingBoundaries.size === 0)
   ) {
```

Comment:
Can we move all this code to the Suspense offscreen instance instead of scheduling a passive effect on the tracing marker?